torch/linalg/__init__.py

```diff
@@ -345,14 +361,28 @@
 """)
 
 eigvals = _add_docstr(_linalg.linalg_eigvals, r"""
-linalg.eigvals(input, *, out=None) -> Tensor
+linalg.eigvals(A, *, out=None) -> Tensor
+
+Computes the eigenvalues of a square matrix.
 
-Computes the eigenvalues of a square matrix. Supports batched inputs.
+Letting :math:`\mathbb{K}` be :math:`\mathbb{R}` or :math:`\mathbb{C}`,
+the eigenvalues of a square matrix :math:`A \in \mathbb{K}^{n \times n}` are defined
+as the roots (counted with multiplicity) of the polynomial of degree `n`
```

Comment:
"of a polynomial \`p\` of degree \`n\` such that"

Reply:
> The polynomial `p` of degree `n`  given by:

Corrected both occurrences